utilities/update_docs_version.sh

```diff
@@ -10,7 +10,7 @@ RELEASED_VERSION=$(mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate
 if [ "${RELEASED_VERSION##*-}" != "SNAPSHOT" ]; then
     echo "Changing version to $RELEASED_VERSION in README files"
     # Get list of directories for which README.md must be updated
-    module_folders=($(find . -maxdepth 1 -name 'gcloud-java*' -type d) .)
+    module_folders=($(find . -maxdepth 2 \( -regex "./gcloud-java[-[a-z]+]+*" -o -regex "./gcloud-java-contrib/gcloud-java[-[a-z]+]+*" \) -type d) . ./gcloud-java)
```

Comment:
I am running:
```bash
module_folders=($(find . -maxdepth 2 \( -regex "./gcloud-java[-[a-z]+]+*" -o -regex "./gcloud-java-contrib/gcloud-java[-[a-z]+]+*" \) -type d) . ./gcloud-java)
echo ${module_folders[*]}
```
And what I get is:
```
. ./gcloud-java
```
For completeness `find . -maxdepth 1 -type d` returns:
```
.
./.git
./.settings
./gcloud-java
./gcloud-java-bigquery
./gcloud-java-contrib
./gcloud-java-core
./gcloud-java-datastore
./gcloud-java-dns
./gcloud-java-examples
./gcloud-java-resourcemanager
./gcloud-java-storage
./src
./target
./utilities		
```